src/core/reducers/abuse.js

```diff
@@ -54,21 +58,26 @@ export function hideAddonAbuseReportUI({
   };
 }
 
-type LoadAddonAbuseReportType = {
+type LoadAddonAbuseReportParams = {|
   addon: {|
     guid: string,
     id: number,
     slug: string,
   |},
   message: string,
   reporter: AbuseReporter | null,
-};
+|};
+
+type LoadAddonAbuseReportAction = {|
+  type: typeof LOAD_ADDON_ABUSE_REPORT,
+  payload: LoadAddonAbuseReportParams,
+|};
 
 export function loadAddonAbuseReport({
   addon,
   message,
   reporter,
-}: LoadAddonAbuseReportType = {}) {
+}: LoadAddonAbuseReportParams): LoadAddonAbuseReportAction {
   if (!addon) {
     throw new Error('addon is required');
   }
```

Comment:
I noticed you updated this below (to use invariants instead) so I am wondering if you could do the same here (and potentially in the other places where it could be used)..?

Reply:
I will file an issue to improve this file, which includes:

- converting `if/Error` to `invariant`
- add more flow types

Reply:
https://github.com/mozilla/addons-frontend/issues/6959